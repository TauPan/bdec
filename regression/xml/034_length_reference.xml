<protocol>
    <sequence name="blah">
        <field name="length total" length="8" type="integer" />
        <field name="length header" length="8" type="integer" />
        <field name="header" length="(${length header} * 8)" type="text" />
        <field name="data" length="((${length total} * 8) - len{header})" type="text" />
    </sequence>

</protocol>
