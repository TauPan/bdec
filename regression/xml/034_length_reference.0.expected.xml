
           <blah>
              <length-total>10</length-total>
              <length-header>6</length-header>
              <header>header</header>
              <data>data</data>
           </blah> 